paddle/fluid/pybind/eager_py_layer.cc

```diff
@@ -108,6 +108,20 @@ PyObject* pylayer_method_name(PyObject* self, PyObject* noargs) {
   EAGER_CATCH_AND_THROW_RETURN_NULL
 }
 
+PyObject* tensor_view(paddle::experimental::Tensor* tensor) {
```

Comment:
Better to change another name tensor view is not fully supported with this method

Reply:
done